src/App.js

```diff
@@ -254,13 +259,9 @@ class App extends React.Component {
 										<div className="user-leftSection-titleContainer">
 											<h2 className="user-leftSection-title">Updates</h2>
 										</div>
-										<p>
-											No class on Tuesday, October 20th. Quiz #3 will be pushed
-											back to Wednesday, and it will contain 10.1 and some of 10.2.
-										</p>
 										<p>
 											Since Dale Hoffman's website is down due to the wind, here
-											is a link to the{" "}
+											is a link to the
```

Comment:
Adding a space back
```suggestion
											is a link to the{" "}
```

Reply:
Forgot to approve this change. New commit: 2c40465d460e74deb31a4883e53a00790d197c0a